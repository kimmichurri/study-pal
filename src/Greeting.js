import React from 'react';

export default function Greeting () {
    return (
        <p className="question-text">Welcome! Choose a category of flashcards to study or visit your study bank to keep practicing.</p>
    )
}