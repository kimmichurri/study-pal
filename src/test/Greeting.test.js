import React from 'react';
import Greeting from '../Greeting';
import { shallow } from 'enzyme';

describe('Greeting', () => {

    let wrapper;
  
    beforeEach(() => {
  
      wrapper = shallow(
        <Greeting />
      )
    });

    it('should match the snapshot', () => {
        expect(wrapper).toMatchSnapshot();
      });
});