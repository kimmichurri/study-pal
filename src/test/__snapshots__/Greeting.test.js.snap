// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Greeting should match the snapshot 1`] = `
<p
  className="question-text"
>
  Welcome! Choose a category of flashcards to study or visit your study bank to keep practicing.
</p>
`;
